pkg/apis/ports.go

```diff
@@ -19,4 +19,6 @@ const (
 	AntreaControllerAPIPort = 10349
 	// AntreaAgentAPIPort is the default port for the antrea-agent APIServer.
 	AntreaAgentAPIPort = 10350
+	// AntreaAgentClusterPort is the default port for the antrea-agent cluster
```

Comment:
I feel we should add more information here about what is antrea-agent cluster.

Reply:
To be consistent with config option, AntreaAgentClusterMembershipPort?

Reply:
sounds good, done